python/tskit/trees.py

```diff
@@ -6599,9 +6603,10 @@ def divergence(
         sets of nodes from ``sample_sets``.
         This is the "average number of differences", usually referred to as "dxy";
         a common citation for this definition is Nei and Li (1979), who called it
-        :math:`\pi_{XY}`. Note that computing the divergence of a population to itself
-        gives the mean pairwise nucleotide diversity within that population,
-        which is :meth:`diversity <.TreeSequence.diversity>`.
+        :math:`\pi_{XY}`. Note that the mean pairwise nucleotide diversity *within* a
+        population is :meth:`diversity <.TreeSequence.diversity>`, which can also be
+        obtained by computing the divergence of a population to itself by using indexes
+        (see the note below).
```

Comment:
```suggestion
        :math:`\pi_{XY}`. Note that the mean pairwise nucleotide diversity of a
        sample set to itself (computed by passing an index of the form (j,j))
        is its :meth:`diversity <.TreeSequence.diversity>` (see the note below).
```

Gah, I hate it how "population" has a habit of sneaking in where it doesn't belong.